app/models/excursion.rb

```diff
@@ -0,0 +1,13 @@
+class Excursion < ApplicationRecord
+  validates_presence_of :title
+  validates_presence_of :description
+  validates_presence_of :location
+
+  has_many :taggings
+  has_many :tags, through: :taggings
+  has_many :favorites
+  validates_presence_of :title
+  belongs_to :user
+
+  alias_attribute :author_id, :user_id
```

Comment:
Is this actually being used anywhere?

Reply:
You're right, removed line 9 to remove repeating line, thanks for checking that out @cunninghamge . Was line 10 or 12 questionable too?